components/skeleton/skeleton-group-mixin.js

```diff
@@ -0,0 +1,28 @@
+import { dedupeMixin } from '@open-wc/dedupe-mixin';
+import { SubscriberRegistryController } from '../../controllers/subscriber/subscriberControllers.js';
+
+export const SkeletonGroupMixin = dedupeMixin(superclass => class extends superclass {
+
+	constructor() {
+		super();
+
+		this._skeletonActiveCached = true;
+		this._skeletonSubscribers = new SubscriberRegistryController(this, 'skeleton', {
+			onSubscribe: this._handleSubscriberChange.bind(this),
+			onUnsubscribe: this._handleSubscriberChange.bind(this),
+			updateSubscribers: this._checkSubscribersReadyToDisplay.bind(this)
+		});
+	}
+
+	_checkSubscribersReadyToDisplay(subscribers) {
```

Comment:
This seems to be using the old terminology, maybe something like `_checkSubscribersSkeletonState`